Library/Homebrew/extend/ENV/std.rb

```diff
@@ -108,7 +108,7 @@ def gcc_4_0
 
   def gcc_4_2
     super
-    set_cpu_cflags
+    set_cpu_cflags "-march=core2 -msse4"
```

Comment:
@mistydemeo @MikeMcQuaid Is this change necessary? Does Apple GCC 4.2 support `-march=native`?

Reply:
Mike says that Apple GCC 4.2 does not support `-march=native`.